pages/initiatives/gsoc/gsoc2021ideas.md

```diff
@@ -235,3 +235,136 @@ OWASP IIDS is an open source software that leverages the benefits of Artificial
 * [Sri Harsha Gajavalli](mailto:sriharsha.g@owasp.org)
 
 ----
+
+
+### [OWASP OWTF](https://github.com/owtf/owtf)
+Offensive Web Testing Framework (OWTF) is a project focused on penetration testing efficiency and alignment of security tests to security standards like the OWASP Testing Guide (v3 and v4), the OWASP Top 10, PTES and NIST. Most of the ideas below focus on rewrite of some major 
+components of OWTF to make it more modular. OWTF is moving to a fresh codebase with a fully Docker testing and deployment environment.
+
+#### OWASP OWTF - Passive Online scanner improvements
+
+##### Brief Explanation
+
+OWTF allows many passive tests, such as those using third party websites like Google, Bing, etc. searches, as well as handy "Search for vulnerability" search boxes (i.e. Fingerprinting plugin). This feature involves the creation of a "script" that produces an interactive OWTF report with the intention of hosting it in the github.io site. The idea here is to have a passive, JavaScript-only interactive report available on the owtf.github.io site, so that people can try OWTF "without installing anything", simply visiting a URL.
+
+This would be a normal OWTF interactive report where the user can:
+* Enter a target
+* Try passive plugins (only the parts that use no tools)
+* Play with boilerplate templates from the OWTF interactive report
+An old version of the passive online scanner is hosted [here](https://owtf.github.io/online-passive-scanner).
+
+##### LEGAL CLARIFICATION (Just in case!): 
+The passive online scanner, simply makes OWTF passive testing through third party websites more accessible to anybody, however it is the user that must 
+1. click the link manually + 
+2. do something bad with that afterwards + 
+3. doing 1 + 2 WITHOUT permission :). 
+Therefore this passive online scanner does not do anything illegal 
+[More information about why this is not illegal here](http://www.slideshare.net/abrahamaranguren/legal-and-efficient-web-app-testing-without-permission) 
+(recommended reading!)
+
+For background on OWASP OWTF please see: [OWASP OWTF](https://www.owasp.org/index.php/OWASP_OWTF)
```

Comment:
Should update these... throughout the OWTF additions

Reply:
Looks like OWASP.org content needs an update it still talks about gsoc 2020 and release 2.3b

Reply:
Added a PR https://github.com/OWASP/www-project-owtf/pulls. Pls have a look!

Reply:
Tks, the links here should still be updated